src/Titon/Route/Route.hh

```diff
@@ -216,7 +216,8 @@ class Route implements Serializable {
         }
 
         $action = $this->getAction();
-        $object = Registry::factory($action['class'], []);
+
+        $object = Depository::getInstance()->make($action['class']);
```

Comment:
Same goes here. The argument are already explicit. We aren't gaining much from using the `Depository`.